hack/util.sh

```diff
@@ -459,9 +459,7 @@ function install_registry() {
 readonly -f install_registry
 
 function wait_for_registry() {
-	local generation="$(oc get dc/docker-registry -o 'jsonpath={.metadata.generation}')"
-	local onereplicajs='{.status.observedGeneration},{.status.replicas},{.status.updatedReplicas},{.status.availableReplicas}'
-	wait_for_command "oc get dc/docker-registry -o 'jsonpath=${onereplicajs}' --config='${ADMIN_KUBECONFIG}' | grep '^${generation},1,1,1$'"  "$((5*TIME_MIN))"
+	wait_for_command "oc rollout status dc/docker-registry --request-timeout=1m"  "$((5*TIME_MIN))"
```

Comment:
> All we need for this helper is to see that the inital deployment of the registry is complete

I was under the impression that @liggitt's point was that we _also_ used this helper to do tests later after changing this component?